robottelo/constants/__init__.py

```diff
@@ -634,7 +634,8 @@
     },
     'rhel8_bos': {
         'id': 'rhel-8-for-x86_64-baseos-rpms',
-        'name': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS RPMs 8',
+        'name': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS RPMs 8.5',
```

Comment:
question, why hardcoding rhel8 repos to 8.5? any reason?

Reply:
Convert2rhel needs to use specific minor version repos.
As I am the only one that is currently using this constant, I think it is OK hardcode those.

Reply:
@peterdragun that's a dangerous assumption to make.

`rhel8_aps` is being used in `tests/foreman/api/test_contentmanagement.py`, and it doesn't look like rhel8_bos is used directly, but the field here could be composed within tests.

I think a much more robust implementation would set this based on `content_host.os_version` within the test.  The update to rhel 8.6 being available (and the default for rhel8 contenthost checkouts) will happen separately from this PR, and will break this assumption in the future.

Reply:
that test in `tests/foreman/api/test_contentmanagement.py` is actually using the kickstart package not that I was changing. And those kickstart repos are locked to 8.5 or 8.4, even version doesn't match between aps and bos repos. So I am really the only one using this repo.

Actually, I wanted to change the version based on the content host version but, then I couldn't find a way to keep the same parametrization that I have created. Currently, all repos for RHEL are synced just once and then they are used 2 times. I will try to find the solution for this, but I couldn't is it okay to create a new constant with name e.g. `rhel85_bos`?